modules/rostests/apitests/user32/GetDCEx.c

```diff
@@ -29,33 +28,33 @@ WndProc(
 static
 ATOM
 RegisterClassHelper(
-    PSTR pszClassName,
+    LPCWSTR pszClassName,
     UINT style,
     WNDPROC pfnWndProc)
 {
-    WNDCLASSA cls;
+    WNDCLASSW cls = { 0 };
 
     cls.style = style;
     cls.lpfnWndProc = pfnWndProc;
     cls.cbClsExtra = 0;
     cls.cbWndExtra = 0;
-    cls.hInstance = GetModuleHandleA(0);
+    cls.hInstance = GetModuleHandleW(0);
     cls.hIcon = 0;
-    cls.hCursor = LoadCursorA(0, IDC_ARROW);
+    cls.hCursor = LoadCursorW(0, MAKEINTRESOURCEW(32512));
```

Comment:
Keep the original line here (or it should still be possible to use IDC_ARROW and keeping LoadCursor without A/W)

Reply:
Ok, I'll edit, I was also wondering whether to revert it, I tried to see if it affected the problematic functions, but it didn't.